js/user-form.js

```diff
@@ -1,17 +1,59 @@
 const MAXIMUM_PRICE = 100000;
 const WHOLESALE_OFFER = '100';
 const ZERO_GUESTS = '0';
+
 const form = document.querySelector('.ad-form');
+const types = form.querySelector('#type');
+const price = form.querySelector('#price');
 const rooms = form.querySelector('#room_number');
 const capacity = form.querySelector('#capacity');
+const timein = form.querySelector('#timein');
+const timeout = form.querySelector('#timeout');
 
 const pristine = new Pristine(form, {
   classTo: 'ad-form__element',
   errorTextParent: 'ad-form__element',
   errorTextClass: 'error-text',
 });
 
-pristine.addValidator(form.querySelector('#price'), (value) => value <= MAXIMUM_PRICE, 'Не больше 100 000 руб.');
+timein.addEventListener('change', () => {
+  timeout.value = timein.value;
+});
+
+timeout.addEventListener('change', () => {
+  timein.value = timeout.value;
+});
+
+const HOUSING_TYPE = {
+  bungalow: 0,
+  flat: 1000,
+  hotel: 3000,
+  house: 5000,
+  palace: 10000
+};
+
+const validatePrice = (value) => {
+  const chosenType = types.querySelector('option:checked');
+
+  return value <= MAXIMUM_PRICE && value >= HOUSING_TYPE[chosenType.value];
+};
+
+const priceErrorMessage = (value) => {
+  const chosenType = types.querySelector('option:checked');
+  if (value >= MAXIMUM_PRICE) {
+    return 'Не больше 100 000 руб.';
+  }
+
+  return `Минимальная цена для данного типа жилья - ${HOUSING_TYPE[chosenType.value]} руб.`;
+};
+
+pristine.addValidator(price, validatePrice, priceErrorMessage);
+
+types.addEventListener('change', () => {
```

Comment:
при изменении типа ты должен и менять атрибут мин и плейсхолдер на инпуте с ценой
```suggestion
type.addEventListener('change', () => {
```